model/src/pyrenew/processes/rtrandomwalk.py

```diff
@@ -0,0 +1,89 @@
+import numpyro as npro
+import numpyro.distributions as dist
+from pyrenew.metaclasses import RandomProcess
+from pyrenew.processes.simplerandomwalk import SimpleRandomWalkProcess
+from pyrenew.transform import AbstractTransform, LogTransform
+
+
+class RtRandomWalkProcess(RandomProcess):
```

Comment:
Given that we now have a `latent` module for latent (but potentially observable) variables, I propose creating an an `Rt` class there. 

To represnt $\mathcal{R}(t)$ varying in time according to a particular stochastic process on a particular transformed scale, one would instantiate the `Rt` class (or a subclass of it), using stochastic processes from `pyrenew.processes`, transformations from `pyrenew.transform` etc.

Reply:
Works for me. Create an issue?